tests/set-labels/empty-list/simple/.expected/results.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 exitCode: 1
 items:
   - image: gcr.io/kpt-fn/set-labels:unstable
-    stderr: '[error]: failed to configure function: `functionConfig` must be either a `ConfigMap` or `SetLabels`'
+    stderr: 'failed to evaluate function: error: function failure'
```

Comment:
Do you know why this does not give "`functionConfig` must be either a `ConfigMap` or `SetLabels`" error?
I think for empty function config, giving "`ConfigMap` or `SetLabels`" is more helpful.

Reply:
I can modify the error message here. Using (true, nil) would cause confusion in fn render, as the function would be marked PASS instead of FAIL